test/VirtualDeviceTest.ts

```diff
@@ -23,16 +23,16 @@ describe("VirtualDevice", function() {
             assert.isDefined(results);
         });
 
-        it.skip("Should have stream URL", async () => {
+        it("Should have stream URL", async () => {
             const sdk = newVirtualDevice();
-            const result = await sdk.message("tell test player to play");
+            const result = await sdk.message("open test player and play");
             console.log("Output: " + JSON.stringify(result));
             assert.isDefined(result.streamURL);
             assert.isTrue((result.streamURL as any).startsWith(
                 "https://feeds.soundcloud.com/stream/309340878-user-652822799-episode-010"));
         });
 
-        it.skip("Should have debug info", async () => {
+        xit("Should have debug info", async () => {
```

Comment:
this is still excluded, is that the intention?

Reply:
https://github.com/bespoken/virtual-device/issues/134 should be fixed to get the test working